source/docs/casper/resources/tutorials/beginner/cep78/index.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Cep78
+slug: /cep78
```

Comment:
Should we use slashes in the slug values to allow us to differentiate and navigate to these URls from the address bar?

Reply:
Do we do it elsewhere?

Reply:
I did not introduce this, but the use seems persistent in other directories as well, so I'd assume it's supposed to stay?

Reply:
We need to a) figure out where this goes and b) put it into the sidebar

Reply:
Slug should have complete path. So, for example instead of localhost:3000/cep78, it should be localhost:3000/resources/tutorials/beginner/cep78

Reply:
I removed the slug for the cep78 index page.